components/inputs/test/input-date.visual-diff.js

```diff
@@ -198,6 +198,7 @@ describe('d2l-input-date', () => {
 							);
 							input.dispatchEvent(e);
 						});
+						await page.waitForTimeout(100);
```

Comment:
I tried modifying this to wait for calendar updateComplete but that didn't work. I haven't seen this test flake since adding this.